### PR TITLE
Fix rest API tests fixtures

### PR DESCRIPTION
**Pull request by engahmeds3ed**
## Description

Just to make our rest API tests working with latest WordPress version 5.7

## Type of change

Please delete options that are not relevant.

- [x] Test Bug fix (non-breaking change which fixes an issue)

## Is the solution different from the one proposed during the grooming?

No need for grooming, this is development team initiative..

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Made tests green again

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by remyperona — APPROVED** [review]

**Review by remyperona — CHANGES_REQUESTED** [review]
Can you update the test workflow to use the latest WP version instead of 5.6.2?

**Comment by engahmeds3ed** [comment]
@Tabrisrp 

Done

File: .github/workflows/test_wprocket.yml [hunk 1]
```diff
@@ -11,16 +11,16 @@ on:
...
-    
+
     strategy:
       fail-fast: false
       matrix:
         operating-system: [ubuntu-latest]
         php-versions: ['7.3', '7.4']
-        wp-versions: ['5.6.2']
-  
+        wp-versions: ['5.7']
```
Comment (remyperona) [inline .github/workflows/test_wprocket.yml]:
```suggestion
        wp-versions: ['latest']
```

**Review by remyperona — CHANGES_REQUESTED** [review]

**Review by remyperona — APPROVED** [review]

**Comment by remyperona** [comment]
You can merge 👍🏼

**Pull request merged**